Plugins/DD4hep/src/DD4hepDetectorSurfaceFactory.cpp

```diff
@@ -43,6 +43,11 @@ void Acts::DD4hepDetectorSurfaceFactory::construct(
   ACTS_DEBUG("Recursive search did yield: "
              << cache.sensitiveSurfaces.size() << " sensitive surface(s), "
              << cache.passiveSurfaces.size() << " passive surface(s)");
+
+  // Check for binning patching
+  if (!cache.binnings.empty() && cache.sExtent.has_value()) {
+    ACTS_DEBUG("Checking for patching the binnig structure ... ");
```

Comment:
```suggestion
    ACTS_DEBUG("Checking for patching the binning structure ... ");
```

Also: what does "patching" mean here?

Reply:
I changed all of this to 'automatic range adaption' because that's what it does.

Reply:
That's a great improvement! Thank you!